core/src/debt.ts

```diff
@@ -0,0 +1,231 @@
+import type { DebtAuctionActive, DebtAuctionTransaction } from './types';
+import BigNumber from './bignumber';
+import getContract from './contracts';
+import { RAD_NUMBER_OF_DIGITS, WAD_NUMBER_OF_DIGITS } from './constants/UNITS';
+import { Contract } from 'ethers';
+import { MKR_NUMBER_OF_DIGITS, RAD, WAD } from './constants/UNITS';
+import memoizee from 'memoizee';
+import getNetworkDate from './date';
+import { CompensationAuctionBase, DebtAuction, Notifier, CompensationAuctionTransactionFees } from './types';
+import { getEarliestDate } from './helpers/getEarliestDate';
+import executeTransaction from './execute';
+import { convertMkrToDai } from './calleeFunctions/helpers/uniswapV3';
```

Comment:
Why was this function used instead of the newly created `convertDaiToMkr`?

Reply:
That's the incautious use of autocomplete from my side. Essentially needs to be fixed. Should i open a quickfix PR?